ddtrace/tracer/textmap.go

```diff
@@ -446,11 +446,19 @@ func (p *propagator) injectTextMap(spanCtx ddtrace.SpanContext, writer TextMapWr
 	if ctx.origin != "" {
 		writer.Set(originHeader, ctx.origin)
 	}
+	var baggageItems []string
 	ctx.ForeachBaggageItem(func(k, v string) bool {
 		// Propagate OpenTracing baggage.
 		writer.Set(p.cfg.BaggagePrefix+k, v)
+		encodedKey := encodeKey(k)
+		encodedValue := encodeValue(v)
+		baggageItems = append(baggageItems, fmt.Sprintf("%s=%s", encodedKey, encodedValue))
```

Comment:
```suggestion
		baggageItem := encodedKey + "=" + encodedValue
		baggageItems = append(baggageItems, baggageItem)
```

`fmt.Sprintf` should be avoided, it's quite slow for this kind of code. It's a string that can be build by concatenation.